spec/amber/cli/commands/generator_spec.cr

```diff
@@ -1,36 +1,82 @@
 require "../../../spec_helper"
+require "../../../support/helpers/cli_helper"
+require "../../../support/fixtures/cli_fixtures"
 
-module Amber::CLI
-  begin
-    describe MainCommand::Generate do
-      context "scaffold" do
-        it "generates and compile generated app" do
-          ENV["AMBER_ENV"] = "test"
-          MainCommand.run ["new", TESTING_APP]
-          Dir.cd(TESTING_APP)
-          MainCommand.run ["generate", "scaffold", "Animal", "name:string"]
-          Amber::CLI::Spec.prepare_yaml(Dir.current)
+include CLIHelper
+include CLIFixtures
 
-          `shards build`
+module Amber::CLI
+  describe MainCommand::Generate do
```

Comment:
Same thing here - we could have `describe "amber generate" do` and remove the context below.